website/server/controllers/api-v3/groups.js

```diff
@@ -911,6 +911,8 @@ function _sendMessageToRemoved (group, removedUser, message, isInGroup) {
  * @apiError (400) {NotAuthorized} onlyLeaderCanRemoveMember Only the group
                                                              leader can remove members.
  * @apiError (400) {NotAuthorized} memberCannotRemoveYourself Group leader cannot remove themselves
+ * @apiError (400) {NotAuthorized} cannotRemoveQuestOwner Group leader cannot remove
```

Comment:
the code here is 401 not 400, I think there was a mistake in the previous docs. NotAuthorized is 401, BadRequest is 400